doc/shared-libraries.md

```diff
@@ -5,6 +5,11 @@ Shared Libraries
 
 The purpose of this library is to make the verification functionality that is critical to Bitcoin's consensus available to other applications, e.g. to language bindings.
 
+Policy correctness as enforced by latest software version is also made available. To dissociate between invalidity reasons, either consensus or standard, an application
```

Comment:
In commit "[doc] Document shared-libraries.md on standard verification" (4ea5e6d18203f1c45fa68b8f5585c7fc96e5677b)

Maybe standardize on standardness: s/correctness/standardness/ and s/standard/standardness/